model/internal/pdata/common.go

```diff
@@ -131,6 +131,35 @@ func NewValueBytes(v []byte) Value {
 	return Value{orig: &otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_BytesValue{BytesValue: v}}}
 }
 
+func newValueFromRaw(iv interface{}) Value {
+	switch tv := iv.(type) {
+	case nil:
+		return NewValueEmpty()
+	case string:
+		return NewValueString(tv)
+	case int64:
```

Comment:
Here I think we should accept all type of `ints` (int8, int32, int, int64) and `uints` (uint8, uint32, uint, uint64)

Reply:
I assume you skipped `int16` and `uint16` unintentionally

Reply:
Added all variants of int also `pdata.Map` and `pdata.Slice` as another representation of map and slice